deps/fdasrsf/DynamicProgrammingQ2.c

```diff
@@ -0,0 +1,37 @@
+#include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "dp_grid.h"
+
+void DynamicProgrammingQ2(double* Q1, double* T1, double* Q2, double* T2,
+                          int m1, int n1, int n2,
+                          double* tv1, double* tv2, int n1v, int n2v, double* G,
+                          double* T, double* size, double lam1)
+{
+    int* idxv1;
```

Comment:
I usually prefer to declare the variables when they are first assigned to, if possible. This can be done in C99 and C11 standards.